core/dbt/task/deps.py

```diff
@@ -37,10 +37,13 @@ def track_package_install(
         self, package_name: str, source_type: str, version: Optional[str]
     ) -> None:
         # Hub packages do not need to be hashed, as they are public
-        # Use the string 'local' for local package versions
+        # Use the string 'local' for local and 'tarball' package versions
```

Comment:
This comment doesn't really add value, you can delete the entire line instead of modifying it.

Reply:
removed with 3d48f6ad612299db7d3d92464d2df1a518f991f7